draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -453,19 +414,150 @@ System  |    +--------+       +--------+       +--------+   | System
     |<---------------------- DetNet MPLS --------------------->|
     |                                                          |
     |<--------------- End to End DetNet Service -------------->|
+
+   -------------------------- Data Flow ------------------------->
+
+    X   = Optional service protection (none, PRF, PREOF, PEF/POF)
+    DFx = DetNet member flow x over a TE LSP
 ]]></artwork>
 </figure>
 
- <t>
-  <xref target="fig_pw_detnet2"/> illustrates how an end to end MPLS-based DetNet
-  service is provided where the end systems are not able to send and receive
-  DetNet flows. In this example, the nodes labeled CE1 and CE2 could be
-  non-DetNet aware IP routers or hosts. Note that E1 and E2 are edge
-  nodes as they sit boundaries of the DetNet enabled domain.
- </t>
 
-<figure align="center" anchor="fig_pw_detnet2"
- title="MPLS-Based DetNet (non-MPLS End System)">
+    <t>
+      [Authors note: TBD - should the rest of this section be cut?]

```

Comment:
No, these are good and straight-fort  description for scenarios where "MPLS to the desktop" is not possible.